docs/source/kit/motor-board.rst

```diff
@@ -0,0 +1,83 @@
+Motor Board
+===========
+
+The Motor Board can be used to control two 12V DC motors. These can be
+used for moving your robot, although don’t feel you are limited to using
+them for this purpose.
+
+The speed and direction of the two outputs are controlled independently
+through the USB interface. The USB interface is isolated from the rest
+of the board to prevent damage to the host in the case of a board
+failure. Due to this isolation the board must have power applied to both
+the power connector (from the 12V outputs on the power board) and the
+USB port. If the board does not have power applied to the power
+connector then the kit will report that there is a problem with the
+motor board.
+
+Board diagram
+-------------
+
+.. figure:: /_static/kit/mcv4b_diagram.png
+   :alt: Board Diagram
+   :scale: 75%
+
+   Board Diagram
+
+Indicators
+----------
+
++---------------+--------------------------------------+---------------+
+| LED           | Meaning                              | Initial       |
+|               |                                      | power-up      |
+|               |                                      | state         |
++===============+======================================+===============+
+| Power         | The board is powered                 | On            |
++---------------+--------------------------------------+---------------+
+| M0/M1         | Brightness indicates speed, colour   | Off           |
+| Speed/Directi | indicates direction                  |               |
```

Comment:
Breaking the word "Direction" across two lines here results in there being an unsightly space in the middle of the word in the rendered page: https://sbot.readthedocs.io/en/latest/kit/motor-board.html#indicators

Reply:
Blame pandoc :angry: Fixed